### PR TITLE
Add Method type export in lambda-alb

### PR DESCRIPTION
**Pull request by epicallan**

### CODE REVIEW

File: lambda-alb/src/AWS/Lambda/ALB.hs [hunk 1]
```diff
@@ -9,40 +9,39 @@ where
...
-import qualified Data.Aeson           as JSON
-import qualified Data.Aeson.Types     as JSON
-import qualified Data.CaseInsensitive as CI
-import qualified Data.HashMap.Strict  as HashMap
-import qualified Network.HTTP.Types   as HTTP
+import qualified Data.Aeson            as JSON
+import qualified Data.Aeson.Types      as JSON
+import qualified Data.CaseInsensitive  as CI
+import qualified Data.HashMap.Strict   as HashMap
+import qualified Network.HTTP.Types    as HTTP
```
Comment (dkubb) [inline lambda-alb/src/AWS/Lambda/ALB.hs:19]: __suggestion (nitpick):__ It looks like unnecessary whitespace was added here.

**Review by dkubb — APPROVED** [review]

File: lambda-alb/src/AWS/Lambda/ALB.hs (hunk 1 above)
Comment (epicallan) [reply lambda-alb/src/AWS/Lambda/ALB.hs:19]: Let me fix it up. Wish we had a linter for it but will try to build muscle memory.
